<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1637</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="4,2">
      <item row="1" column="0">
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
      <item row="2" column="0">
       <widget class="QTableWidget" name="tableActivities">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::DashLine</enum>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Tipo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Titolo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ora inizio</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ora fine</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Frequenza</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QComboBox" name="comboType">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="maxVisibleItems">
           <number>2</number>
          </property>
          <item>
           <property name="text">
            <string>Evento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Attivit√†</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editTitle"/>
        </item>
        <item row="1" column="1">
         <widget class="QTimeEdit" name="timeStart"/>
        </item>
        <item row="3" column="1">
         <widget class="QTimeEdit" name="timeEnd">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboFrequency">
          <item>
           <property name="text">
            <string>Nessuna</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Giornaliera</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Settimanale</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mensile</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="btnSaveTask">
          <property name="text">
           <string>Salva </string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="btnDeleteTask">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Elimina </string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="btnUpdateTask">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Aggiorna </string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkEndTime">
          <property name="text">
           <string>Ora di fine</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="textFrequency">
          <property name="text">
           <string>Frequenza: </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
